changelog.d/implicit-rules-sc-core.added

```diff
@@ -0,0 +1 @@
+Changed `semgrep-core` so that it can now be run with `-rules` on `.yaml` files which do not have a top-level `rules: ...` key. This means you can now copy paste from the playground editor directly into a `.yaml` file for use with `semgrep-core`.
```

Comment:
hmm the absence of toplevel rules: was already accepted I think (see the comment in the code).
But you had to have an array, so something like

- id: rule1
  message: xxx
  ...
- id: rule2
  message: yyy
  ...

What you added is when you have only 1 rule, to not even have to put it in an array with the leading - but just
id: rule1
message: bla
...